loadtest/control/simulcontroller/actions.go

```diff
@@ -1604,12 +1614,11 @@ func (c *SimulController) updateThreadRead(u user.User) control.UserActionRespon
 			return control.UserActionResponse{Err: control.NewUserError(errors.New("updateThreadRead: current team should be set"))}
 		}
 		threads, err := u.GetUserThreads(team.Id, &model.GetUserThreadsOpts{
-			PageSize:   25,
-			Extended:   false,
-			Deleted:    false,
-			Unread:     false,
-			Since:      0,
-			TotalsOnly: false,
+			PageSize: 25,
+			Extended: false,
+			Deleted:  false,
+			Unread:   false,
+			Since:    0,
```

Comment:
Is the omission of both `TotalsOnly` and `ThreadsOnly` here intentional? (Feels like the pattern above is to be explicit.)

Reply:
Yep, intentional. This is an edge case - we make this API call only when the simulated user's threads aren't already loaded, so we need to fetch both the counts and the threads. 

I omitted both since they need to be false (which is the default). I can see why explicit would be better. I'll add them back in.